app/views/DeclaredAnswersView.scala.html

```diff
@@ -44,5 +44,17 @@ <h2 class="govuk-heading-m">@messages("declaredAnswers.trn", trn)</h2>
 
     <p>@messages("declaredAnswers.p1", declarationSent)</p>
 
-    @govukSummaryList(SummaryList(rows = formatSummarySections(entities)))
+    @for(section <- entities){
+        @if(section.headingKey.isDefined){
+            <h2 class="govuk-heading-l">@messages(section.headingKey.get)</h2>
+        }
```

Comment:
Rather than doing a `.isDefined` and then `.get` I believe the following is valid markup
```
section.headingKey.map { hk =>
  <h2 class="govuk-heading-l">@messages(hk)</h2>
}
```

Reply:
Updated